app/javascript/components/public_health/PatientsTable.js

```diff
@@ -170,13 +193,14 @@ class PatientsTable extends React.Component {
 
   handleTabSelect = tab => {
     localStorage.removeItem(`SaraPage`);
+    localStorage.removeItem(`SaraSortField`);
```

Comment:
here's another one

Reply:
Per [Kylie's response to a previous thread](https://github.com/SaraAlert/SaraAlert/pull/685#discussion_r569524205), this line will remain. 

I did notice that the call to setState() in this function never actually resets the sort settings (which is important if the user sorts a column in one tab, then goes to another tab that doesn't have that column). So in the next commit, I will update the setState() call so it resets the sort settings.